regulations/static/regulations/js/source/views/preamble-view.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+var $ = require('jquery');
+var _ = require('underscore');
+var Backbone = require('backbone');
+Backbone.$ = $;
+
+var ChildView = require('./main/child-view');
+var CommentView = require('./comment-view');
+
+var PreambleView = ChildView.extend({
+  el: '#content-wrapper',
```

Comment:
I may not be understanding the Backbone stuff correctly, but wouldn't this make a PreambleView for _any_ `#content-wrapper`, many of which aren't related to preambles? Should it be something like
`#content-wrapper[data-page-type=preamble-section]`?